_posts/2022-11-22-10.0.1-release.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title: "Apache Arrow 10.0.1 Release"
+date: "2022-11-22 00:00:00"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+
+The Apache Arrow team is pleased to announce the 10.0.1 release.
+This is mostly a bugfix release that includes [**30 resolved issues**][1]
+from [**xx distinct contributors**][2]. See the Install Page to learn how to
+get the libraries for your platform.
+
+The release notes below are not exhaustive and only expose selected highlights
+of the release. Many other bugfixes and improvements have been made: we refer
+you to the [complete changelog][3].
+
+## Columnar Format Notes
+
+## Arrow Flight RPC notes
+
+## C++ notes
+
+## Go notes
```

Comment:
@zeroshade any notes for Go we should add to the release blog post for 10.0.1?

Reply:
```suggestion
## Go notes
* Added option to support dictionary deltas with IPC ([ARROW-18326](https://issues.apache.org/jira/browse/ARROW-18326))
* Fix dictionary replacement during IPC stream ([ARROW-18317](https://issues.apache.org/jira/browse/ARROW-18317))
* Fix StructBuilder premature release fields ([ARROW-18274](https://issues.apache.org/jira/browse/ARROW-18274))
```
@zeroshade does this seem ok?

Reply:
Thanks @raulcd that looks good to me. Sorry for not responding earlier, I was out at the end of last week for Thanksgiving and forgot to update this yesterday.

Reply:
Thanks @zeroshade !